woocommerce/woocommerce-ios
# Log in: Splash screen is not legible for large font sizes

Fix #911 

This PR is not related to any potential work on the sign in flow. 

The login prologue (the first screen in the login flow) did not support dynamic type properly, so for users that had activated their accessibility font sizes, it could look like this:

<img src= "https://user-images.githubusercontent.com/2722505/58612879-a98a1a80-82e6-11e9-91fe-adf4ed3eb5d4.png" width="350" />

The solution implemented in order to accommodate larger fonts is similar to the one we followed on the WordPress iOS app: make fonts in this screen have a maximum size. That way, fonts can be bigger, and we still maintain a certain layout. It's not perfect, but it's better than nothing.

Now, the LoginPrologue would look like this for the default font size:
<img src="https://user-images.githubusercontent.com/2722505/58613105-506eb680-82e7-11e9-8eb1-135e5c20dc97.png" width="350" />

Like this for the largest font size (without accessibility fonts activated):
<img src="https://user-images.githubusercontent.com/2722505/58613117-5f556900-82e7-11e9-9d64-19f749cdb443.png" width="350" />

And this would be the largest possible accessibility font:
<img src="https://user-images.githubusercontent.com/2722505/58613134-6aa89480-82e7-11e9-8dff-f95fca02535f.png" width="350" />

## Changes
- Encapsulate fonts used in the LoginPrologue into the StyleManager
- Make those fonts support dynamic type, but with a maximum font size.

## Testing
- Checkout the branch, navigate to the LoginPrologue
- Change the device (or the simulator) font size. On the simulator, the font size can be changed using Xcode's Accessibility Inspector (Xcode > Open Developer Tool > Accessibility Inspector)
- Notice if the labels and button change fonts and are readable.

Update release notes:
